### ISSUE TITLE
ADVZ PayloadProver support requests that span multiple polynomial

### ISSUE THREAD

**Issue by ggutoski** [EspressoSystems/jellyfish#424]
# tl;dr

ADVZ PayloadProver currently fails if a proof request spans multiple polynomials. Remove this restriction.

https://github.com/EspressoSystems/jellyfish/blob/265eaaa059e0a501824e5aadf7ac5f32d9e4a6b8/primitives/src/vid/advz/payload_prover.rs#L380-L385

# Background

Distilled from [zulip](https://espresso.zulipchat.com/#narrow/stream/330967-Sequencer/topic/Namespace.20boundaries.20in.20jellyfish):

- In the past we vaguely intended that each namespace would correspond to 1 polynomials in the ADVZ payload partition. In this case the sequencer must take into account these polynomial boundaries when it builds the block payload bytes. ie. sequencer must leave zero padding between namespaces so that each namespace starts a new ADVZ polynomial.
- Complication: how to handle small namespaces? The vague intuition is to pack many small namespaces into a single polynomial. I guess it would be the sequencer who decides which namespaces get packed and which get their own polynomial.
- It's hard to do this without breaking abstraction: an abstract VID scheme doesn't know anything about polynomial boundaries.
- A hacky solution: defining an abstract notion of "alignment" in a general VID scheme. So we could have `next_alignment_boundary(index) -> Option<usize>`. A VID scheme that doesn't care about alignment can just return `None`, ADVZ can return the next polynomial boundary.
- Alternative: abandon the rule of 1 namespace per polynomial and just accept the fact that jellyfish namespace proofs might not align well with polynomial boundaries.

## Conclusion

Ignore polynomial boundaries for now. We'll revisit later if we run into perf issues.
